client/js/renderPreview.js

```diff
@@ -47,7 +47,7 @@ function renderPreview(preview, msg) {
 	container.trigger("keepToBottom");
 }
 
-$("#chat").on("click", ".toggle-button", function() {
+$("#chat").on("click", ".text .toggle-button", function() {
```

Comment:
This makes sure that the thumbnail toggle handler does not react to condensed toggle clicks. It is possible because `text` class is now absent from the condensed template.